plugins/@grouparoo/google-sheets/src/lib/sheet-import/sourcePreview.ts

```diff
@@ -0,0 +1,21 @@
+import Spreadsheet from "./spreadsheet";
+import {
+  App,
+  Source,
+  SimpleAppOptions,
+  SimpleSourceOptions,
+  plugin,
+} from "@grouparoo/core";
+
+// TODO: be nice to have signature of what this should return
+// pretty sure it's array of objects with header keys
```

Comment:
Yep! Here's an example https://github.com/grouparoo/grouparoo/blob/46b1a470f9f47f4437a2781ebe5fbe22a2764bdc/core/api/__tests__/utils/specHelper.ts#L195-L199

Reply:
was more of a documentation or typescript idea. thank, though.